snipsmanager/commands/run.py

```diff
@@ -169,6 +170,44 @@ def handle_intent(self, intent, payload=None):
                 else:
                     getattr(skill, intent_def.action)()
 
+    def handle_dialogue_events_async(self, state):
+        """ Handle the dialogue API events."""
+        if(state == self.server.DIALOGUE_EVENT_STARTED):
+            state_name = "session_started"
+        elif(state == self.server.DIALOGUE_EVENT_ENDED):
+            state_name = "session_ended"
+        elif(state == self.server.DIALOGUE_EVENT_QUEUED):
+            state_name = "session_queued"
+
```

Comment:
To cover our bases, I would add an error if for some reasons in the future we add a new dialogue event and forget to handle it here :
```
else:
    raise NotImplementedError(
            'Dialogue event unrecognised, please update handle_dialogue_events_async in run.py')
```